draft-ietf-quic-transport.md

```diff
@@ -2703,6 +2706,8 @@ ensures that connections are not closed after new activity is initiated.
 
 To avoid excessively small idle timeout periods, endpoints MUST increase the
 idle timeout period to be at least three times the current Probe Timeout (PTO).
+This allows for multiple PTOs to expire prior to idle timeout, allowing for at
+least one packet to be lost prior to idle timeout.
```

Comment:
Allow for recovery, not loss.

```suggestion
This allows for multiple PTOs to expire prior to idle timeout, allowing for the endpoint
to recover any packets lost during this time.
```

Reply:
Perhaps "ensuring a single packet loss event cannot cause an idle timeout"? We certainly can't recover from *any* degree of packet loss.

Reply:
I like your suggestion, taken.